rubygem-foreman_virt_who_configure/rubygem-foreman_virt_who_configure.spec

```diff
@@ -0,0 +1,102 @@
+%{?scl:%scl_package rubygem-%{gem_name}}
+%{!?scl:%global pkg_name %{name}}
+
+%global gem_name foreman_virt_who_configure
+%global plugin_name virt_who_configure
+
+Name: %{?scl_prefix}rubygem-%{gem_name}
+Version: 0.0.1
+Release: 1%{?foremandist}%{?dist}
+Summary: A plugin to make virt-who configuration easy
+Group: Applications/Systems
+License: GPLv3
+URL: https://github.com/theforeman/foreman_virt_who_configure
+
+Source0: http://rubygems.org/gems/%{gem_name}-%{version}.gem
+Requires: foreman >= 1.11
+Requires: %{?scl_prefix_ruby}ruby(release)
+Requires: %{?scl_prefix_ruby}ruby 
+Requires: %{?scl_prefix_ruby}ruby(rubygems) 
+#Requires: %{?scl_prefix}rubygem(deface) 
+Requires: %{?scl_prefix}rubygem(katello) >= 3.0
+BuildRequires: foreman-assets
+BuildRequires: foreman-plugin >= 1.11
+#BuildRequires: %{?scl_prefix}rubygem(deface) 
+BuildRequires: %{?scl_prefix}rubygem(katello) >= 3.0
+BuildRequires: %{?scl_prefix_ruby}ruby(release)
+BuildRequires: %{?scl_prefix_ruby}ruby 
+BuildRequires: %{?scl_prefix_ruby}rubygems-devel 
+BuildArch: noarch
+Provides: %{?scl_prefix}rubygem(%{gem_name}) = %{version}
+Provides: foreman-plugin-%{plugin_name}
+
+%description
+A plugin to make virt-who configuration easy.
+
+
+%package doc
+Summary: Documentation for %{pkg_name}
+Group: Documentation
+Requires: %{?scl_prefix}%{pkg_name} = %{version}-%{release}
+BuildArch: noarch
+
+%description doc
+Documentation for %{pkg_name}.
+
+%prep
+%{?scl:scl enable %{scl} - << \EOF}
+gem unpack %{SOURCE0}
+%{?scl:EOF}
+
+%setup -q -D -T -n  %{gem_name}-%{version}
+
+%{?scl:scl enable %{scl} - << \EOF}
+gem spec %{SOURCE0} -l --ruby > %{gem_name}.gemspec
+%{?scl:EOF}
+
+%build
+# Create the gem as gem install only works on a gem file
+%{?scl:scl enable %{scl} - << \EOF}
+gem build %{gem_name}.gemspec
+%{?scl:EOF}
+
+# %%gem_install compiles any C extensions and installs the gem into ./%%gem_dir
+# by default, so that we can move it into the buildroot in %%install
+%{?scl:scl enable %{scl} - << \EOF}
+%gem_install
+%{?scl:EOF}
```

Comment:
Does foreman_virt_who_configure have any c extensions? If not you can simplify this line to prep with an example from https://github.com/theforeman/foreman-packaging/pull/1537/files